include/oboe/AudioStream.h

```diff
@@ -544,6 +542,27 @@ class AudioStream : public AudioStreamBase {
         return mPerformanceHintEnabled;
     }
 
+    /**
+     * If you have called setPerformanceHintEnabled(true) then you can give
+     * the performance manager more information about your workload.
+     * You can call this at the beginning of the callback when you figure
+     * out what your workload will be.
+     *
+     * The workload can be in your own units. If you are synthesizing music
```

Comment:
Can you give a suggested range or some best practices? What happens if they use 1000 for each voice?